internal/DateComponentPicker/DateComponentPicker.js

```diff
@@ -55,10 +55,12 @@ const DateComponentPickerBase = kind({
 	render: ({children, max, value, ...rest}) => (
 		<Picker
 			{...rest}
+			css={css}
 			index={value}
 			max={max}
 			min={0}
 			orientation="vertical"
+			showSecondaryValues
```

Comment:
`showSecondaryValues` is actually a prop, but it had a default value of `true` and I forgot to update the sampler with the new prop. (that's why `Gallium` also had 2 sets of values)
In order to fix this, I modified the code so the 'secondary sets of values' only appear when `Silicon` skin is used. What do you think of this approach?

Reply:
> In order to fix this, I modified the code so the 'secondary sets of values' only appear when Silicon skin is used. What do you think of this approach?

Sounds good to me.

> showSecondaryValues is actually a prop, but it had a default value of true
My opinion is that we need to set `showSecondaryValues` to true at this line if the skin is equal to `silicon`.
```
showSecondaryValues={skin === 'silicon'}
```

Additional comment:
If possible, I think, it would be better to provide the pro defining the number of the values in a picker. It would be 3 by default and it would be 5 in `silicon` skin. If you have any concern, you could ignore this additional comment.